File: wrappers/dotnet/indy-sdk-dotnet/WalletApi/WalletAccessFailedException.cs
namespace Hyperledger.Indy.WalletApi
{
    /// <summary>
    /// Exception thrown when attempting to open a wallet using invalid credentials.
    /// </summary>
    public class WalletAccessFailedException : IndyException
    {
        const string message = "The wallet could not be opened because invalid credentials were provided.";

        /// <summary>
        /// Initializes a new WalletAccessFailedException.
        /// </summary>
        internal WalletAccessFailedException() : base(message, (int)ErrorCode.WalletAccessFailed)
        {

        }
    }

}

File: wrappers/android/src/main/java/org/hyperledger/indy/sdk/wallet/WalletAccessFailedException.java
package org.hyperledger.indy.sdk.wallet;

import org.hyperledger.indy.sdk.ErrorCode;
import org.hyperledger.indy.sdk.IndyException;

/**
 * Attempt to open encrypted wallet with invalid credentials
 */
public class WalletAccessFailedException extends IndyException
{
	private static final long serialVersionUID = 3294831240096535507L;
	private final static String message = "The wallet security error.";

	/**
	 * Initializes a new WalletAccessFailedException.
	 */
	public WalletAccessFailedException() {
		super(message, ErrorCode.WalletAccessFailed.value());
	}
}